tools/restatectl/src/commands/log/find_tail.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use anyhow::Context;
+use cling::prelude::*;
+use restate_cli_util::_comfy_table::{Cell, Table};
+use restate_cli_util::ui::console::StyledTable;
+use tonic::codec::CompressionEncoding;
+
+use restate_admin::cluster_controller::protobuf::cluster_ctrl_svc_client::ClusterCtrlSvcClient;
+use restate_admin::cluster_controller::protobuf::{FindTailRequest, TailState};
+use restate_cli_util::c_println;
+
+use crate::app::ConnectionInfo;
+use crate::util::grpc_connect;
+
+use super::LogIdRange;
+
+#[derive(Run, Parser, Collect, Clone, Debug)]
+#[cling(run = "find_tail")]
+pub struct FindTailOpts {
+    /// The log id(s) to find its tail.
+    #[arg(required = true)]
+    log_id: Vec<LogIdRange>,
+}
+
+async fn find_tail(connection: &ConnectionInfo, opts: &FindTailOpts) -> anyhow::Result<()> {
+    let channel = grpc_connect(connection.cluster_controller.clone())
+        .await
+        .with_context(|| {
+            format!(
+                "cannot connect to cluster controller at {}",
+                connection.cluster_controller
+            )
+        })?;
+    let mut client =
+        ClusterCtrlSvcClient::new(channel).accept_compressed(CompressionEncoding::Gzip);
+
+    let mut chain_table = Table::new_styled();
+    let header_row = vec![
+        "LOG-ID",
+        "SEGMENT",
+        "FROM-LSN",
+        "TO-LSN",
```

Comment:
I'm not sure about printing FROM-LSN here. The command (from its name) is focused on the tail of the log, FROM-LSN here only zooms to the last segment in the log which is implicit and will be confusing to users (and me :))